addons/jr/config.cpp

```diff
@@ -1,14 +1,5 @@
 #include "script_component.hpp"
 
-#define private 0       // hidden
```

Comment:
This was the only one used in one place, I removed those defines, should be fine?